tests/playwright/pages/CheckoutPageSFRA5.mjs

```diff
@@ -85,16 +86,16 @@ export default class CheckoutPageSFRA5 {
 
   isPaymentModalShown = async (imgAltValue) => {
     await expect(this.paymentModal.locator(`img[alt='${imgAltValue}']`))
-      .toBeVisible({ timeout: 10000 });
+      .toBeVisible({ timeout: 15000 });
   }
 
   navigateToCheckout = async (locale) => {
     await this.page.goto(this.getCheckoutUrl(locale));
   };
 
-  goToCheckoutPageWithFullCart = async (locale) => {
-    await this.addProductToCart(locale);
-    await this.successMessage.waitFor({ visible: true, timeout: 10000 });
+  goToCheckoutPageWithFullCart = async (locale, itemCount = 1) => {
+    await this.addProductToCart(locale, itemCount);
+    await this.successMessage.waitFor({ visible: true, timeout: 15000 });
```

Comment:
Can we move timeout to a variable for all CheckoutPages?

Reply:
Potentially we can, and I thought about it. However certain actions may need different timeouts, hence static usage. In test config file we have different timeouts set for actions, assertions and the total test run. The places we have overriden this are preventing early failures.

I agree that we can improve the whole approach to timeouts, especially after working with resourceful runners, but I'd rather handle it with care, after having run the tests with the current setting

Reply:
we can possibly set a variable for default timeout time, and unless passed a different value we use the default